Common/CommonTypes.h

```diff
@@ -30,7 +30,7 @@ struct u24_be {
 	}
 };
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
I think _MSC_VER is more correct here.

-[Unknown]

Reply:
True, I will change that.